sdk/rust/oak/src/lib.rs

```diff
@@ -394,6 +400,7 @@ pub fn random_get(buf: &mut [u8]) -> Result<(), OakStatus> {
 pub fn result_from_status<T>(status: i32, val: T) -> Result<T, OakStatus> {
     match OakStatus::from_i32(status) {
         Some(OakStatus::Ok) => Ok(val),
+        Some(OakStatus::ErrInvalidChannel) => Ok(val),
```

Comment:
Please add a comment to explain why an error is treated as ok, future "us" will be thankful for it :)

Reply:
This is really an ugly hack, and I don't like it. I think `wait_on_channels` should not return the status in an error. It should always return `Ok` plus an indicator to distinguish between `ReadReady` and `HasInvalid`, and only return an error if something like termination has happened.